libs/vkd3d/state.c

```diff
@@ -1050,6 +1052,9 @@ static HRESULT d3d12_root_signature_init_local(struct d3d12_root_signature *root
         goto fail;
     }
 
+    root_signature->binding_count = info.binding_count;
```

Comment:
Even though static samplers are unsupported right now on this path for us, I think they are meant to be.

So maybe just set that count from the desc here anyway so we don't forget in the future? :)

Reply:
Oh right, I completely forgot about static samplers in local RS. :V Can just add a FIXME ...

Reply:
Fixed.